### ISSUE TITLE
PostObjectLoader breaks when a post is assigned to a non-existent user

### ISSUE THREAD

**Issue by jasonbahl** [wp-graphql/wp-graphql#883]
When querying a list of posts, and a post included in the Loader is assigned to a user that has been deleted, an error with the UserLoader causes the entire PostObjectLoader to bust. 

For example, the following query: 

```
{
  posts {
    nodes {
      id
      title
      date
    }
  }
}
```

If the any of the first 10 posts are set to have a `post_author` ID of a non-existent user, no posts will be returned, and an error will be returned: 

```
{
      "debugMessage": "Argument 1 passed to WPGraphQL\\Model\\User::__construct() must be an instance of WP_User, bool given, called in /var/www/html/wp-content/client-mu-plugins/plugins/wp-graphql/src/Data/Loader/UserLoader.php on line 70",
      "message": "Internal server error",
      "category": "internal",
      "locations": [
        {
          "line": 3,
          "column": 5
        }
      ],
      "path": [
        "posts",
        "nodes",
        0
      ]
    },
```

The culprit of this is that the PostObjectLoader makes a Batch request to first load all users so that they're available to WordPress when `setup_postdata()` is called, but the UserLoader assumes _all_ keys passed to it are existing users. 

We need to support the case of a non-existent user ID being passed to the UserLoader without breaking.

**Comment by jasonbahl** [comment]
@kellenmace I feel like we addressed this, eh? I can look into it again, but I feel like one of your PRs dealt with this bug?

**Comment by jasonbahl** [comment]
closed by #1012
